openupgrade_scripts/scripts/sale_purchase/16.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,9 @@
+---Models in module 'sale_purchase'---
+---Fields in module 'sale_purchase'---
+sale_purchase / product.template         / service_to_purchase (boolean) : not stored anymore
+# DONE: post-migration, convert to company-dependent field
```

Comment:
For better reading (as this file doesn't have syntax highlighting, on contrary than on Python, where I recommend to not put empty lines inside methods due to this), please put an empty line in between each block:

```suggestion
# DONE: post-migration, convert to company-dependent field

```